source/robots.txt

```diff
@@ -1,5 +1,4 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
 
 User-agent: *
-
-Sitemap: https://unboxed.co/sitemap.xml
+disallow: *
```

Comment:
Does this not need to be capitalized? at least that's what the docs seem to suggest:

![image](https://user-images.githubusercontent.com/9452321/127460434-55b106d2-ba48-41f2-8640-25ad6e26c888.png)

https://developers.google.com/search/docs/advanced/robots/create-robots-txt?hl=en